project/src/test/java/com/google/sps/OpportunitySignupDataTest.java

```diff
@@ -0,0 +1,90 @@
+package com.google.sps;
+
+import com.google.sps.data.OpportunitySignup;
+import com.google.sps.data.VolunteeringOpportunity;
+import com.google.sps.servlets.OpportunitySignupDataServlet;
+import com.google.sps.utilities.CommonUtils;
+import com.google.sps.utilities.SpannerClient;
+import com.google.sps.utilities.SpannerTasks;
+import com.google.sps.utilities.SpannerTestTasks;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.util.Arrays;
+import java.util.HashSet;
+import javax.servlet.ServletContextEvent;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mockito;
+import org.springframework.mock.web.MockServletContext;
+
+
+/** Test that tests getting the signups for a volunteering opportunity. */
+@RunWith(JUnit4.class)
+public final class OpportunitySignupDataTest {
+  private static final String NAME = "Performer";
+  private static final int NUMBER_OF_SPOTS = 240;
+  private static final String OPPORTUNITY_ID = "opportunity-id";
```

Comment:
nit: Name this to specify that is it a parameter rather than the actual ID

Reply:
Done - agreed, this adds clarity.